messenger.rst

```diff
@@ -900,6 +900,39 @@ serializer          How to serialize the final payload  ``Redis::SERIALIZER_PHP`
                     ``Redis::OPT_SERIALIZER`` option)
 ==================  =================================== =======
 
+.. tip::
+
+    At some point you'll likely want to scale the number of workers working on your queue.
+    Make sure you assign the correct ``consumer`` and ``group`` values in that case.
+    The more likely case is that you want every worker to work on the queue independently, reducing
```

Comment:
Maybe we should here add  some dsn examples to make it more clear?

> Worker 1: redis://localhost:6379/messages/symfony/consumer-1
> Worker 2: redis://localhost:6379/messages/symfony/consumer-2
> 
> This avoids that the same message is consumed by a parallel running worker
> ...
> ...
> ...
> If you want that another consumer consumes also all messages you need to 
> configure it for another group 
>
> Worker 1: redis://localhost:6379/messages/group-1/consumer-1
> Worker 2: redis://localhost:6379/messages/group-2/consumer-1

Reply:
Done!